admin-dev/themes/new-theme/js/pages/order/view/order-product-edit.js

```diff
@@ -45,13 +45,13 @@ export default class OrderProductEdit {
 
   setupListener() {
     this.quantityInput.on('change keyup', (event) => {
-      const quantity = Number(event.target.value);
+      this.quantity = Number(event.target.value);
```

Comment:
Thanks for the PR,
I am not sure if it is needed to add  `quantity` as attribute in this class, but only keep the constant for this temporary use. If else, I think this change should also be done in `order-product-add.js`

Reply:
@arouiadib it is because of https://github.com/PrestaShop/PrestaShop/blob/1.7.7.x/admin-dev/themes/new-theme/js/pages/order/view/order-product-edit.js#L98, that's mandatory to update the price accordingly.
On `order-product-add.js` on the other hand, it's handled differently so it's not needed.